docs/en/CSMM/advanced-feature-guide-fallingblocks.md

```diff
@@ -72,6 +72,40 @@ Our resulting command for the hook is:
 
-Finally we will set up a Custom Discord notification to alert admins about the activity. Under the server settings > Discord settings, click the Custom Notifications button. Choose a channel to send the alert to, and in the "Text to search for" box enter "You have been detected drop mining". As the pm sent to the player includes this text, the player's name, and the coordinates, the notification sent to discord will also include this information in case further action is deemed necessary.
+Finally we will set up a Custom Discord notification to alert admins about the activity.
+
+### Method 1 - Discord Custom Notification
+
+Under the server settings > Discord settings, click the Custom Notifications button. Choose a channel to send the alert to, and in the "Text to search for" box enter "You have been detected drop mining". As the pm sent to the player includes this text, the player's name, and the coordinates, the notification sent to discord will also include this information in case further action is deemed necessary.
+
+### Method 2 - sendDiscord() function
+
+And one more hook, a logLine hook which will search for the text "You have been detected drop mining". As we previously did for our first hook, we will need to pull a couple of variables from this line and reuse them, so first put a dummy command `wait(1)` in the command box.
+
+Add a variable to pick up the player's name. We'll call this variable `miner`. Using the regex `([^"]*)(?=, You have been detected)` we can identify the text between the first `"` and the first `,` in the pm command.
+
+Add a variable to pick up the location. We'll cal this variable `location`. Using the regex `(?<=drop mining at )(-?\d{1,4}, -?\d{1,4})` we can identify the pair of 1-4 digit numbers following the text `drop mining at`.
+
+Finally to send a discord notification we will use the `sendDiscord()` function.
+
+:::tip
+For details on how to identify the channel and role IDs, see [Functions](/en/CSMM/functions.html). 
+:::
+
+For the purpose of this example, our #alerts channel ID is 718425258742527034 and the role we want to alert (@admins) has the ID <@&710992520425246426>. For your discord server these will be different.
+
+The `sendDiscord()` function takes two arguments, a destination channel and a message. The message can include any variables, and we can include discord IDs to mention roles or users. To let the admins know about the drop mining detection, this hook's command will be:
+
+```
+sendDiscord(718425258742527034,"<@&710992520425246426> - \{\{\{custom.miner\}\}\} has been detected drop mining at \{\{custom.location\}\}. Please investigate.")
+```
+
+:::warning
+Because player names often include special characters like apostrophes, we need to use triple curly braces \{\{\{ \}\}\}. By default, handlebars turns these special characters into code-safe HTML entities like \&\#27;. Using triple braces will pass through special characters unchanged to keep the discord message readable.
```

Comment:
```suggestion
Because player names often include special characters like apostrophes, we need to use triple curly braces `\{\{\{ \}\}\}`. By default, handlebars turns these special characters into code-safe HTML entities like \&\#27;. Using triple braces will pass through special characters unchanged to keep the discord message readable.
```